packages/batch-submitter/src/exec/run-batch-submitter.ts

```diff
@@ -21,8 +21,6 @@ const log = new Logger({
   name,
   sentryOptions: {
     release: `batch-submitter@${process.env.npm_package_version}`,
-    // @ts-ignore stackAttributeKey belongs to PinoSentryOptions, not exported
-    stackAttributeKey: 'err',
```

Comment:
I think we still need this, otherwise the default "error" object key is `trace`

Reply:
@snario ok to add this to `Logger` as a whole? then we don't need the linter comment